### ISSUE TITLE
The × character of popup's close button should be explicitly hidden from ATs

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#7472]
## Motivation

Similar to https://github.com/Leaflet/Leaflet/issues/7433, even if the popup's close button had an accessible name (https://github.com/Leaflet/Leaflet/issues/7399) screen readers would still announce "times" or "multiplication" (depending on screen reader and/or browser combination) if the user hovers the close button.

## Proposed solution

The × character (`&#215;`) should be wrapped in `<span aria-hidden="true">` to prevent screen readers from announcing it.

https://github.com/Leaflet/Leaflet/blob/b52961aa469dc17b3710ef7a9c0f13b6f4dba550/src/layer/Popup.js#L194

**Comment by Falke-Design** [comment]
@Malvoz after integrating #7399 the code looks like: 
```
closeButton.innerHTML = '<span role="button" aria-label="Close popup">&#215;</span>';
```

and now you suggest to add a second `<span>` or does it work to have `aria-hidden` and `aria-label` in the same element?
```
closeButton.innerHTML = '<span aria-hidden="true" role="button" aria-label="Close popup">&#215;</span>';
```

**Comment by Malvoz** [comment]
@Falke-Design Thanks for looking into accessibility issues, much appreciated! 🙏🏼

> after integrating #7399 the code looks like:
>
> `closeButton.innerHTML = '<span role="button" aria-label="Close popup">&#215;</span>';`

https://github.com/Leaflet/Leaflet/issues/7399 suggests to add `role` and `aria-label` on the `<a>` element. I [commented](https://github.com/Falke-Design/Leaflet/commit/efbe5c42aa47986bce0b6e43fb40f43d0ce1a0e3#r60616941) on the commit in your fork.

> does it work to have `aria-hidden` and `aria-label` in the same element

No, `aria-hidden="true"` hides the content of that element and everything inside from screen readers, that's why we need the `<span>` as described in this issue.
